packages/client/tests/functional/extensions/itx.ts

```diff
@@ -230,39 +229,50 @@ testMatrix.setupTestSuite(
       ).resolves.not.toThrow()
     })
 
-    test('middleware exclude from transaction also works with extended client', async () => {
-      const xprisma = prisma.$extends({})
+    // This test can lead to a deadlock on SQLite because we start a write transaction and a write query outside of it
+    // at the same time, and completing the transaction requires the query to finish. This leads a SQLITE_BUSY error
+    // after 5 seconds if the transaction grabs the lock first. For this test to work on SQLite, we need to expose
+    // SQLite transaction types in transaction options and make this transaction DEFERRED instead of IMMEDIATE.
```

Comment:
Do we have a Linear task for that?

Reply:
no, it would be a full blown feature request, not a chore task